src/commands/postDeploy/checkLinuxWebAppDownDetector.ts

```diff
@@ -58,18 +57,32 @@ export async function checkLinuxWebAppDownDetector(correlationId: string, node:
                 return undefined;
             }
 
-            // tslint:disable-next-line: no-unsafe-any
-            detectorErrorMessage = await getLinuxDetectorError(context, linuxContainerStartFailureId, node, deployResultTime, node.root.client.fullName);
+            // time constraint of being less than the current time by 14.5 minutes is enforced system wide for all detectors
+            // however LinuxLogViewer detector ignores it and gets the most recent data
+            const nowTime: Date = new Date();
+            const startTimeDate: Date = new Date(nowTime.getTime() - (60 * 60 * 1000));
+            const endTimeDate: Date = new Date(nowTime.getTime() - (30 * 60 * 1000));
+
+            // need to remove the miliseconds from the ISOstring for request
+            const startTimeArray: string[] = startTimeDate.toISOString().split(':');
+            const startTime: string = `${startTimeArray[0]}:${startTimeArray[1]}`;
+
+            const endTimeArray: string[] = endTimeDate.toISOString().split(':');
+            const endTime: string = `${endTimeArray[0]}:${endTimeArray[1]}`;
+
+            detectorErrorMessage = await getLinuxDetectorError(context, linuxLogViewer, node, startTime, endTime, deployment.endTime);
 
             if (!detectorErrorMessage) {
-                // poll every minute
-                await delay(1000 * 60);
+                // poll every 10 seconds
+                await delay(1000 * 10);
             }
         }
 
         if (showOutput) {
-            await ext.ui.showWarningMessage(detectorErrorMessage, { title: 'Open in Portal' });
-            await openInPortal(node.root, `${node.root.client.id}/troubleshoot`, { queryPrefix: `websitesextension_ext=asd.featurePath%3Ddetectors%2F${linuxContainerStartFailureId}` });
+            ext.outputChannel.appendLog(detectorErrorMessage);
+            const openInPortalMsg: string = localize('openInPortal', 'Open in Portal for more details.');
+            await ext.ui.showWarningMessage(`${detectorErrorMessage} ${openInPortalMsg}`, { title: 'Open in Portal' });
```

Comment:
How about we get rid of "Open in Portal for more details" from the message and just make the button "View details"

The duplicate "Open in Portal" bothers me and I don't think we have to tell them beforehand that it'll be in the portal